src/index.ts

```diff
@@ -87,6 +87,7 @@ const initExpress = (): void => {
       cookie: {
         httpOnly: true,
         secure: process.env.NODE_ENV === 'production',
+        expires: new Date(Date.now() + (30 * 86400 * 1000)),
```

Comment:
Can we either configure the `MongoStore` to expire sessions after 30 days (preferable) or drop this to 14 days to line up? Weird to ask clients to keep around invalid cookies for two extra weeks

Reply:
I also realize - isn't this setting Expires based on the time at server initialization? So we'll just be handing out cookies that expire exactly two weeks from when we booted up, which may be several months before the current time?